4-storage/relational-database-shredder/project/assembly.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
```

Comment:
Interesting. But I guess we don't need this duplicate anyway? Newer SBT assembly listed in `plugins.sbt`.